tsdb/head_bench_test.go

```diff
@@ -79,6 +86,74 @@ func BenchmarkHeadStripeSeriesCreate_PreCreationFailure(b *testing.B) {
 	}
 }
 
+func BenchmarkHead_WalCommit(b *testing.B) {
+	seriesCounts := []int{100, 1000, 10000}
+	series := genSeries(10000, 10, 0, 0)
+	histograms := genHistogramSeries(10000, 11, 0, 119, 1, false)
+
+	for _, seriesCount := range seriesCounts {
+		b.Run(fmt.Sprintf("%d series", seriesCount), func(b *testing.B) {
+			for _, samplesPerAppend := range []int64{1, 2, 5, 100} {
```

Comment:
This is only ever 1 when using Prometheus code.

Reply:
can you elaborate a little bit more @bboreham ?
I'm not familiar with this part of the code 🙏🏽